sdk/fist/Azure.ResourceManager.IoTFirmwareDefense/README.md

```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure IoTFirmwareDefense management client library for .NET
+
+**[Describe the service briefly first.]**
```

Comment:
Could we include some introduction of your RP here?

Reply:
Hi @ArcturusZhang the readme has been updated, and we added tests as well. Can you review again please? thanks!